--- rstart-1.0.4/Makefile.am.orig	2010-10-30 23:36:38.000000000 +0200
+++ rstart-1.0.4/Makefile.am	2010-10-31 08:10:53.471723445 +0100
@@ -24,9 +24,10 @@
 rstart_serverdir = $(libdir)/X11/rstart
 rstart_server_PROGRAMS = rstartd.real
 
+configdir = $(libdir)/X11/rstart
 AM_CFLAGS = $(CWARNFLAGS) $(RSTART_CFLAGS)		\
 		-DSERVERNAME=\"rstart\"			\
-		-DDEFAULT_CONFIG=\"$(libdir)/X11/rstart/config\"
+		-DDEFAULT_CONFIG=\"$(configdir)/config\"
 
 rstartd_real_LDADD = $(RSTART_LIBS)
 
@@ -51,7 +52,6 @@
 
 # config data
 
-configdir = $(libdir)/X11/rstart
 config_DATA = config
 
 config: config.cpp
