--- rstart-1.0.2/Makefile.am.orig	2006-10-15 15:27:26.815305250 +0200
+++ rstart-1.0.2/Makefile.am	2006-10-15 15:27:36.831931250 +0200
@@ -136,14 +136,14 @@
 	done;								\
 	);								\
 									\
-	cd $(DESTDIR)$(DATA_DIR)/contexts;				\
-	sed -e '/^$$/d' -e '/^#/d' @Aliases |				\
+	abssrcdir="`pwd`" ; cd $(DESTDIR)$(DATA_DIR)/contexts;      \
+	sed -e '/^$$/d' -e '/^#/d' $${abssrcdir}/contexts/@Aliases |    \
 	    while read real aliases; do					\
 	        for i in $$aliases; do					\
 	            $(RM) $$i;						\
 	            $(LN) $$real $$i;					\
-	            $(RM) $(DESTDIR)$(rstart_serverdir)/commands/$$i;	\
-	            $(LN) $$real $(DESTDIR)$(rstart_serverdir)/commands/$$i; \
+				$(RM) $(DESTDIR)$(DATA_DIR)/commands/$$i;   \
+				$(LN) $$real $(DESTDIR)$(DATA_DIR)/commands/$$i; \
 	        done;							\
 	    done;							\
 	$(RM) @Aliases
